website/src/data/users.tsx

```diff
@@ -498,11 +498,16 @@ const Users: User[] = [
     source: 'https://github.com/pamelafox/django-quiz-app',
     tags: ['python', 'django', 'webapps', 'appservice', 'bicep', 'azuredb-postgreSQL', 'keyvault'],
   },
-
-  /*
-  Pro Tip: add your site in alphabetical order.
-  Appending your site here (at the end) is more likely to produce Git conflicts.
-   */
+  {
+    title: 'FastAPI on Azure Container Apps',
+    description:
+      'A simple JSON API using the Python FastAPI app framework, designed for deployment on Azure Container Apps. Includes auto-configuration of gunicorn with uvicorn worker count based on CPUs available.',
+      preview: require('./images/simple-fastapi-container.png'),
```

Comment:
I added [simple-fastapi-container.png](https://github.com/Azure/awesome-azd/pull/125/files#diff-b3b5b78b450a31d144e26b69c8020345b4dc9f3c5ea97c4695ba8119f150e782) in this PR. Its the same image currently as the flask api image, but I thought it better to copy in case they diverged.